File: Data Structures & Algorithms/number-of-senior-citizens/submission-0.java
class Solution {
    public int countSeniors(String[] details) {
        int ans = 0;
        for(String i : details) if(((int)(i.charAt(11) - '0'))*10 + (int)(i.charAt(12) - '0') > 60) ans+=1;
        return ans;
    }
}

File: Data Structures & Algorithms/first-missing-positive/submission-0.cpp
class Solution {
public:
    int firstMissingPositive(vector<int>& nums) {
        unordered_map<int,bool>mp;
        for(int i : nums) if(i > 0) mp[i] = true;
        int ans = 1;
        while(mp.count(ans)) ans+=1;
        return ans;
    }
};